specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/wsn/server.may.send.extended.pong.control.frames/request.rpt

```diff
@@ -0,0 +1,35 @@
+connect tcp://127.0.0.1:8001
```

Comment:
This is not testing server functionality. Rather it is testing that the client can receive an extended PONG without falling over. I suggest renaming this test to "client.should.receive.extended.pong.frame"